pkg/kubernetes/client.go

```diff
@@ -19,6 +19,7 @@ import (
 
 var (
 	loadTestMasterLabelSelector = "app=loadtest-master"
+	loadTestWorkerLabelSelector = "app=loadtest-worker-pod"
```

Comment:
where is this being used?

Reply:
Now it's used for both JMeter and Locust backends to specify the worker pods.